docs/breaking/registration.md

```diff
@@ -0,0 +1,53 @@
+# Project Registration Update
+Sphinx is updating the way that you register projects. Instead of specifying your Gnosis Safe owners, threshold and salt nonce in your Solidity scripts, you will now register your projects via the Sphinx UI. In your script, you will simply specify the name of your project.
+
+This change is intended to improve the UX of using Sphinx and to open the door for us to implement other feature requests such as sharing Gnosis Safes between Sphinx projects and importing Gnosis Safes that were deployed using alternative interfaces (such as the Gnosis Safe UI) into Sphinx.
+
+## Update Guide
+These steps will walk you through the process of getting upgraded to the new version. It should only take a few minutes for you to complete.
+
+### 1. Update your Sphinx Plugin and Sphinx Solidity Library
+
+Update the Sphinx plugin version by bumping the version in your `package.json` file to the latest version `0.32.0`.
+
+Then update the Sphinx Solidity library by running `sphinx install`:
+```
+npx sphinx install
+```
+
+### 2. Configure the `SPHINX_ORG_ID` environment variable
+Sphinx now requires your org id to be defined as an environment variable instead of in your scripts. You'll want to update your `.env` file to include it.
+```
+SPHINX_ORG_ID=<your org id>
+```
+
+> If you are using Sphinx in CI, you will also need to define the `SPHINX_ORG_ID` environment variable in your CI process.
+
+### 3. Fetch your `sphinx.lock` file and commit it to version control
+Sphinx now stores information about your projects in a `sphinx.lock` file. This is an autogenerated file. You should not modify it manually. Whenever you register a new project with Sphinx, you'll want to update your `sphinx.lock` file.
```

Comment:
I agree; the user will likely forget to do this. I think it's acceptable b/c it's only an issue for the deploy CLI command or if your tests utilize sphinx. If you're proposing, then everything will work correctly. 

I've also [designed the error messages to account for this scenario](https://github.com/sphinx-labs/sphinx/pull/1667#discussion_r1578281344).

Reply:
If you think it would be helpful, I could add a prompt to the website telling the user to run `sphinx sync` in their repo after they've registered a project.